cmd/scheduler/cmd/scheduler.go

```diff
@@ -177,7 +177,7 @@ func Run(ctx context.Context, options *scheduleroptions.Options) error {
 	kubeSharedInformerFactory.WaitForCacheSync(ctx.Done())
 	kcpSharedInformerFactory.WaitForCacheSync(ctx.Done())
 	edgeSharedInformerFactory.WaitForCacheSync(ctx.Done())
-	c.Start(ctx, 1)
+	go es.Run(ctx, 1)
```

Comment:
Is there a need to wait here for that `es.Run(..)` to finish, lest the overall `main` finish too soon?

Reply:
Added waiting here.